### DISCUSSION TITLE
joypads with double VID/PID

### DISCUSSION THREAD

**Discussion by texita** [o0Zz/sys-con#20] [Q&A]
I got 2 hori battle pad, if i connect them to the pc or the switch with sys-con i generally get this: [0f0d-00dc] for this ID the buttons mapping configuration is already present in the default config.ini.

On the switch with sys-con (discovery mode 0) is happening a thing, if i connect a pro controller and/or the other hori battle pad, the second hori battle pad get this [0f0d-00c1] (loving the auto add feature that helped me a lot because this behavior was driving me crazy at the beginning) with this VID/PID the controller is recognized as a pro controller/generic (switch mode?) and has to add a new configuration to the config file to have this controller working:

[0f0d-00c1] ;Hori battlepad (switch mode)
b=2
a=3
x=4
y=1
l=7
r=8
zl=5
zr=6
minus=9
plus=10
capture=14
home=13
lstick_click=11
rstick_click=12

in this way the controller has even the capture feature enabled.

Hope this help others with this types of controllers.

**Comment by o0Zz** [comment]
Thanks for this information, it's sound's like [0f0d-00dc] refer to the vid/pid for xbox360 driver while [0f0d-00c1] seems to refer to the switch version. I'm not sure to understand why this controller change its mode when you already have a prcontroller plugged but that's fine, I think you analyze is correct and I will add this mapping to the master branch, Thanks
